tests/components/google_travel_time/test_sensor.py

```diff
@@ -0,0 +1,242 @@
+"""Test the Google Maps Travel Time sensors."""
+
+from unittest.mock import patch
+
+import pytest
+
+from homeassistant.components.google_travel_time.const import (
+    CONF_ARRIVAL_TIME,
+    CONF_DEPARTURE_TIME,
+    CONF_TRAVEL_MODE,
+    DOMAIN,
+)
+
+from .const import MOCK_CONFIG
+
+from tests.common import MockConfigEntry
+
+
+@pytest.fixture(name="mock_update")
+def mock_update_fixture():
+    """Mock an update to the sensor."""
+    with patch("homeassistant.components.google_travel_time.sensor.Client"), patch(
+        "homeassistant.components.google_travel_time.sensor.distance_matrix",
+        return_value={
+            "rows": [
+                {
+                    "elements": [
+                        {
+                            "duration_in_traffic": {
+                                "value": 1620,
+                                "text": "27 mins",
+                            },
+                            "duration": {
+                                "value": 1560,
+                                "text": "26 mins",
+                            },
+                            "distance": {"text": "21.3 km"},
+                        }
+                    ]
+                }
+            ]
+        },
+    ):
+        yield
```

Comment:
If you return the mocked method from the fixture, other mocks can just update the return_value. That way it's easier to use it in different places. 
```suggestion
    ) as mock_distance_matrix:
        yield mock_distance_matrix
```

Reply:
Done